### ISSUE TITLE
Consider shipping a ponyfill, or the ability to override the native version

### ISSUE THREAD

**Issue by bschlenk** [github/ariaNotify-polyfill#63]
Hi! I've been testing this polyfill across different browsers and screen readers. What I've found is that the polyfill works really well, but now that the latest Chrome and FireFox ship `ariaNotify` natively, the polyfill isn't always active. This only seems to cause an issue with JAWS, which just does not work with the native `ariaNotify`, but does work with the polyfill. For consistency across browsers, I'd like to be able to enable the polyfill even if ariaNotify is present. Or even better, expose a ponyfill as well so we can call that directly and not have to overwrite the native method.
